examples/iam-connector/config.env

```
@@ -0,0 +1,6 @@
+SYNQLY_ORG_TOKEN=<synqly_org_token>
+USER_EMAIL=<user_email>
+OKTA_URL=<okta_url>
+OKTA_TOKEN=<okta_token>
+OKTA_CLIENT_ID=<okta_client_id>
```

to be consistent, let's do

```suggestion
SYNQLY_ORG_TOKEN=
USER_EMAIL=
OKTA_URL=
OKTA_TOKEN=
OKTA_CLIENT_ID=
```